internal/txlib/add.go

```diff
@@ -47,13 +50,27 @@ func validateFileFilter(input string) error {
 	if res != 1 {
 		return errors.New("you need one <lang> in your File Filter")
 	}
+	if len(filepath.Ext(input)) < 2 {
+		return errors.New("you need to add an extension to your file")
+	}
 	return nil
 }
 
 func validateSourceFile(input string) error {
 	if len(input) < 1 {
 		return errors.New("you need to add a Source File")
 	}
+
+	if len(filepath.Ext(input)) < 2 {
+		return errors.New("you need to add an extension to your Source File")
+	}
+
+	curDir, err := os.Getwd()
+	_, err = os.Stat(filepath.Join(curDir, input))
+
+	if err != nil {
+		return errors.New("you need to add a Source File that exists")
+	}
```

Comment:
You can use [`os.ErrNotExist`](https://pkg.go.dev/os@go1.17.5#pkg-variables)

```suggestion
	if err != os.ErrNotExist {
		return errors.New("you need to add a Source File that exists")
	}
```